build.gradle

```diff
@@ -121,11 +155,21 @@ repositories {
 }
 
 dependencies {
+    implementation 'junit:junit:4.13.2'
+    implementation 'org.junit.jupiter:junit-jupiter:5.8.2'

```

Comment:
btw is there a reason why we promoted `org.junit.jupiter:junit-jupiter:5.8.2` to `implementation`? Whats up with `junit:junit:4.13.2`?

Reply:
Not sure what happened with this change, I must've messed up merging from the MDK or gotten careless with IntelliJ's suggested changes. I reverted this entirely.